watson/cli.py

```diff
@@ -478,6 +478,9 @@ def edit(watson, id):
     The `$EDITOR` environment variable is used to detect your editor.
     """
     if not id:
+        if not len(watson.frames):
+            click.echo("No frame to edit. It's time to start a new one!")
+            return
```

Comment:
I think click returns 0 when you return from a command, 1 when you raise, and 2 when it fails at parsing arguments. Do you think we should raise?